docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -30,6 +30,21 @@ depending on the desired :ref:`retry policy <fte-retry-policy>`.
 
     retry-policy=QUERY
 
+.. note::
+
+  Support for fault-tolerant execution of SQL statements varies on a
+  per-connector basis:
+
+  * Fault-tolerant execution of :ref:`read operations <sql-read-operations>` is
+    supported by all connectors.
+  * Fault tolerant execution of :ref:`write operations <sql-write-operations>`
+    is only supported by the :doc:`/connector/delta-lake`,
+    :doc:`/connector/hive`, and :doc:`/connector/iceberg`.
+
+  Note that setting ``retry-policy`` effectively disables :ref:`write operations
```

Comment:
This should be a warning imho and remove the "Note that" and maybe also do not quote the error message specifically

Reply:
Switched to warning and reworded a bit to emphasize the "warning" part of this note.

I think the error message text is important to put here, because when troubleshooting people are going to search for that text. The text also does not mention fault-tolerant execution making it harder to trace the cause back here.

We'll just have to update the text when it changes, business as usual.